README.md

```diff
@@ -36,6 +36,21 @@ To get the application working locally for development, make sure to create a fi
 ENV=dev
 ```
 
+To have the whole setup running, including the [timed-frontend](https://github.com/adfinis-sygroup/timed-frontend), we need some more strings attached.
+Add the `timed.local` entries to your host file:
+```bash
+echo "127.0.0.1 timed.local" | sudo tee -a /etc/hosts
+```
+
+Start the required services for a fully-fledged local development setup:
+```bash
+docker-compose --file docker-compose.optional.yml up -d
```

Comment:
I don't see that file anywhere...

Reply:
Decided to include everything in the `docker-compose.yml` since a OIDC provider is needed anyways.